frontend/components/PlatformCompatibility/PlatformCompatibility.tsx

```diff
@@ -48,6 +48,31 @@ const displayIncompatibilityText = (err: Error) => {
   }
 };
 
+// const tipContent = (
+//   <>
+//     Estimated compatibility based on the <br />
+//     tables used in the query. Querying <br />
+//     iPhones, iPads, and Android hosts is not <br />
+//     supported.
+//   </>
+// );
+
+// TODO(android): replace with the above tipContent when Android feature flag is removed
```

Comment:
Rather than add a feature flag check for this tiny copy change (which would require some needless prop drilling), I'm thinking we can just leave the old tooltip for now.